src/styles/createTypography.spec.js

```diff
@@ -34,4 +34,10 @@ describe('createTypography', () => {
     const typography = createTypography(palette, { htmlFontSize: 10 });
     assert.strictEqual(typography.display4.fontSize, '11.2rem');
   });
+  
+  it('should create a typography with custom display4', () => {
+    const customFontSize = '18px';
+    const typography = createTypography(palette, { display4: { fontSize: customFontSize } });
+    assert.strictEqual(typography.display4.fontSize, customFontSize);
+  });
```

Comment:
This test is fine, but it doesn't really relate to this PR, since it's a runtime (JS) test. What we need is something that checks for compilation failures, see [here](https://github.com/mui-org/material-ui/blob/f69c7b52c72b213af460f863e0a65ccbc470b2c6/test/typescript/styles.spec.tsx#L94).

Reply:
Sorry, misread the name of the file you wanted me to add the test to.